vmm/src/vm_config.rs

```diff
@@ -645,4 +796,94 @@ pub struct VmConfig {
     // valid, and will be closed when the holding VmConfig instance is destroyed.
     #[serde(skip)]
     pub preserved_fds: Option<Vec<i32>>,
+    #[serde(default)]
+    pub landlock_enable: bool,
+    pub landlock_config: Option<Vec<LandlockConfig>>,
```

Comment:
Please keep the member as `landlock_rules` but the type can be `LandlockConfig` or `LandlockRulesConfig` if you want to be consistent since we don't have `net_config`. The command line option should also remain `--landlock-rules`.

Reply:
Ok. I renamed `landlock_config` to `landlock_rules` instead. Now the usage is consistent everywhere.